src/transforms/concat.js

```diff
@@ -91,6 +91,24 @@ module.exports = (code, exports, moduleMap) => {
         path.replaceWith(t.identifier(`$${mod}$exports`));
       }
     },
+    MemberExpression(path) {
+      if (!path.isReferenced()) {
+        return;
+      }
+
+      let {object, property} = path.node;
+      if (!t.isIdentifier(object) || !t.isIdentifier(property)) {
+        return;
+      }
+
+      let match = object.name.match(EXPORTS_RE);
+      if (match) {
+        let exportName = '$' + match[1] + '$export$' + property.name;
+        if (path.scope.hasBinding(exportName)) {
+          path.replaceWith(t.identifier(exportName));
+        }
+      }
+    },
```

Comment:
This means code like this:

```javascript
import * as test from './foo';

console.log(test.b);
```

will compile to:

```javascript
var $4$export$b = 1;

console.log($4$export$b);
```

No namespace object needed!

Reply:
Awesome!